beangulp/importers/csv_test.py

```diff
@@ -177,6 +185,25 @@ def test_date_formats(self, filename):
 
         """, entries)
 
+    @test_utils.docfile
+    def test_one_line_no_header(self, filename):
+        """\
+          11/7/2016,A,2
+        """
+        file = cache.get_file(filename)
+        importer = csv.Importer({Col.DATE: 0,
+                                 Col.NARRATION: 1,
+                                 Col.AMOUNT: 2},
+                                'Assets:Bank', 'EUR', [],
+                                dateutil_kwds={'dayfirst': True},
```

Comment:
Please keep the tests as focused as possible to the aspect they want to tests. In other words, there is no need to test the `dateutil_kwds` argument here.